app/templates/gulpfile.babel.js

```diff
@@ -0,0 +1,326 @@
+                return;
+            }
+            clearInterval(appReadyInterval);
+            serverReady = true;
+            cb();
+        }),
+        100);
+}
+
+/********************
+ * Reusable pipelines
+ ********************/
+
+let lintClientScripts = lazypipe()<% if(filters.coffee) { %>
+    .pipe(plugins.coffeelint)
+    .pipe(plugins.coffeelint.reporter);<% } else { %>
+    .pipe(plugins.jshint, 'client/.jshintrc')
+    .pipe(plugins.jshint.reporter, 'jshint-stylish');<% } %>
+
+let lintServerScripts = lazypipe()<% if(filters.coffee) { %>
+    .pipe(plugins.coffeelint)
+    .pipe(plugins.coffeelint.reporter);<% } else { %>
+    .pipe(plugins.jshint, 'server/.jshintrc')
+    .pipe(plugins.jshint.reporter, 'jshint-stylish');<% } %>
+
+let styles = lazypipe()
+    .pipe(gulp.src, paths.client.mainStyle)
+    .pipe(plugins.sourcemaps.init)<% if(filters.stylus) { %>
+    .pipe(plugins.stylus, {
+        use: [nib()],
+        errors: true
+    })<% } if(filters.sass) { %>
+    .pipe(plugins.sass)<% } %>
+    .pipe(plugins.autoprefixer, {browsers: ['last 1 version']})
+    .pipe(plugins.sourcemaps.write, '.')
+    .pipe(gulp.dest, '.tmp/app');<% if(filters.babel || filters.coffee) { %>
+
+let transpile = lazypipe()
+    .pipe(plugins.sourcemaps.init)<% if(filters.babel) { %>
+    .pipe(plugins.babel)<% } else { %>
+    .pipe(plugins.coffee, {bare: true})<% } %>
+    .pipe(plugins.sourcemaps.write, '.');<% } %>
+
+/********************
+ * Tasks
+ ********************/
+
+gulp.task('inject', cb => {
+    runSequence(['inject:js', 'inject:css'], cb);
+});
+
+gulp.task('inject:js', () => {
+    return gulp.src(paths.views.main)
+        .pipe(plugins.inject(gulp.src(_.union(
+            paths.client.scripts,
+            ['!client/**/*.spec.<%= scriptExt %>']
+        ), {read: false}), {
+            starttag: '<!-- injector:js -->',
+            endtag: '<!-- endinjector -->',
+            transform: (filepath) => '<script src="' + filepath.replace('/client/', '') + '"></script>'
+        }))
+        .pipe(gulp.dest('client'));
+});
+
+gulp.task('inject:css', () => {
+    return gulp.src(paths.views.main)
+        .pipe(plugins.inject(gulp.src('/client/**/*.css', {read: false}), {
+            starttag: '<!-- injector:css -->',
+            endtag: '<!-- endinjector -->',
+            transform: (filepath) => '<link rel="stylesheet" href="' + filepath.replace('/client/', '').replace('/.tmp/', '') + '">'
+        }))
+        .pipe(gulp.dest('client'));
+});
+
+// TODO: other styles
+gulp.task('inject:sass', () => {
+    return gulp.src('client/app/app.scss')
+        .pipe(plugins.inject(gulp.src(_.union(paths.client.styles, ['!' + paths.client.mainStyle]), {read: false}), {
+            starttag: '// injector',
+            endtag: '// endinjector',
+            transform: (filepath) => '@import \'' + filepath.replace('/client/app/', '').replace('/client/components/', '../components/') + '\';'
```

Comment:
use as this ``` '@import \'' + filepath.replace('/client/app/', '').replace('/client/components/', '') + '\';' ``` and in style lazypipe as
 ``` .pipe(plugins.sass, {
      includePaths:[
        'client/bower_components',
        'client/app',
        'client/components'
      ]
    })```

then it will be compatible with gruntfile